File: rb/utility/make_context.rb
# BerufsbildungDatenExporte SDK utility: make_context
require_relative '../core/context'
module BerufsbildungDatenExporteUtilities
  MakeContext = ->(ctxmap, basectx) {
    BerufsbildungDatenExporteContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BerufsbildungDatenExporte SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BerufsbildungDatenExporteMakeContext
{
    public static function call(array $ctxmap, ?BerufsbildungDatenExporteContext $basectx): BerufsbildungDatenExporteContext
    {
        return new BerufsbildungDatenExporteContext($ctxmap, $basectx);
    }
}
